lambda/README.md

```diff
@@ -0,0 +1,236 @@
+# Using AWS Lambda with Amazon Aurora DSQL
+
+This document describes how to use AWS Lambda to access Aurora DSQL. In this sample, a CDK application is created with
+a DSQL cluster, and a Lambda function that accesses the cluster to create a table, read and write values to that table, and then
+finally delete the table. The sample code is using Node.js, but it could be created similarly in other languages as well.
+
+## Pre-requisites
+
+- Permissions to create DSQL clusters, AWS Lambdas, and IAM policies
+- You must have installed npm `v8.5.3` or higher.
+- You must have installed aws-cdk `v2.1018.0` or higher.
```

Comment:
This AWS CDK version is incorrect as the latest version as of today is `v2.200.1`.